db/1.9/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -42,6 +42,23 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
+== Snowflake JDBC Driver Throws Exception for Stored Procedure
+
+Some drivers have their own specifications when calling stored procedures. When you set the callable statement with curly brackets `{` and `}` for the Snowflake JDBC driver, for example:
```

Comment:
Running the application when you set the callable statement with curly brackets ({ and }), for example, `{call doubleMyInt(:myInt)}`, causes the Snowflake JDBC driver to throw the following exception: